new/collapse-mixlib-install-in-tk.md

```diff
@@ -0,0 +1,171 @@
+---
+RFC: unassigned
+Title: Collapse Mixlib Install Code Paths in Test Kitchen
+Author: Patrick Wright <patrick@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Collapse Mixlib Install Code Paths in Test Kitchen
+Test Kitchen currently supports two code paths for configuring, downloading and installing Chef and ChefDK on test instances. Both code paths use Mixlib Install but are implemented differently. In the past year, significant effort has been put into Mixlib Install to make it the canonical source for interacting with Chef Software Inc's release infrastructure and generating bootstrap installation scripts (install.sh and install.ps1).
+
+Mixlib Install's new API is currently used for downloads.chef.io and omnitruck.chef.io. It's also used for the chef-ingredient cookbook. Chef transparently implemented the new API and selected config options in Kitchen for testing purposes. It has proven valuable and easy to use. The feature set is limited at this time.
+
+## Motivation
+```
+As a Kitchen User,
+I want to use simple, clear configuration options,
+so that I can setup Kitchen config files with less complexity and obfuscation.
+```
+
+```
+As a Kitchen Developer,
+I want to maintain a single Mixlib Install code path,
+so that I can benefit from the latest API changes and reduce maintenance risk.
```

Comment:
State "reduce maintenance risk" as a positive gain. Maybe something like "consolidate support and maintenance efforts"?